### DISCUSSION TITLE
Creating elementary OS Developer Community

### DISCUSSION THREAD

**Discussion by pablojimpas** [elementary/edw#41] [Ideas]
This idea takes direct inspiration from [UWP Community](https://uwpcommunity.com/) in the Windows world.

It consists of creating a space where independent developers can discuss the state of the platform and get help from other devs or users can directly communicate with devs.

Also, indie devs can get together to collaborate on their apps not only through code but also via chat, so new developers can learn faster.

It will also be cool to make this edw an annual event where this group of indie developers launch their projects.

Speaker: TBD (someone willing to delve into this topic creating this group and leading the way for the developer community, preferably several).

**Comment by JeysonFlores** [comment]
Do you mean something like the Elementary E3? xD

**Comment by cassidyjames** [comment]
@pablojimpas are you suggesting more of a workshop/"BoF" type session at edw?
